docs/cloudflare-infrastructure-guide.md

```
@@ -410,205 +443,266 @@ Turnstile is Cloudflare's CAPTCHA alternative. It protects forms from bots witho
-### Google OAuth "redirect_uri_mismatch" Error
+### OAuth `redirect_uri_mismatch`
 
-**Cause:** The Google Cloud Console does not list the deployment URL as an authorized redirect URI.
+**Cause:** The Google or GitHub OAuth app does not list the deployment URL as an authorized redirect URI.
 
-**Fix:** In the Google Cloud Console, add the redirect URI:
+**Fix:** Add to the OAuth app:
 
-```text
-https://ywcc-capstone.pages.dev/api/auth/callback/google
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)